content/References/ZFSPrimer.md

```diff
@@ -1,179 +1,216 @@
+If an SSD is dedicated as a cache device, it is known as an [L2ARC](https://www.brendangregg.com/blog/2008-07-22/zfs-l2arc.html). 
+Additional read data is cached here, which can increase random read performance. 
+L2ARC does not reduce the need for sufficient RAM. 
+In fact, L2ARC needs RAM to function. 
+If there is not enough RAM for an adequately-sized ARC, adding an L2ARC does not increase performance. 
+Performance actually decreases in most cases, potentially causing system instability. 
 RAM is always faster than disks, so always add as much RAM as possible before considering whether the system can benefit from an L2ARC device.
 
-When applications perform large amounts of *random* reads on a dataset small enough to fit into L2ARC, read performance can be increased by adding a dedicated cache device.
-SSD cache devices only help if the active data is larger than system RAM but small enough that a significant percentage fits on the SSD.
-As a general rule, L2ARC should not be added to a system with less than 32 GiB of RAM, and the size of an L2ARC should not exceed ten times the amount of RAM.
-In some cases, it may be more efficient to have two separate pools: one on SSDs for active data, and another on hard drives for rarely used content.
+When applications perform large amounts of random reads on a dataset small enough to fit into L2ARC, you can increase read performance by adding a dedicated cache device. 
+SSD cache devices only help if the active data is larger than system RAM but small enough that a significant percentage fits on the SSD. 
+As a general rule, do not add L2ARC to a system with less than 32 GiB of RAM, and the size of an L2ARC should not exceed ten times the amount of RAM. 
+In some cases, it might be more efficient to have two separate pools: one on SSDs for active data, and another on hard drives for rarely used content. 
 After adding an L2ARC device, monitor its effectiveness using tools such as `arcstat`.
-To increase the size of an existing L2ARC, stripe another cache device with it.
-The web interface will always stripe L2ARC, not mirror it, as the contents of L2ARC are recreated at boot.
-Failure of an individual SSD from an L2ARC pool will not affect the integrity of the pool, but may have an impact on read performance, depending on the workload and the ratio of dataset size to cache size.
-Note that dedicated L2ARC devices cannot be shared between ZFS pools.
+To increase the size of an existing L2ARC, stripe another cache device with it. 
+The web interface always stripes L2ARC, it does not mirror it, as the contents of L2ARC are recreated at boot. 
+Failure of an individual SSD from an L2ARC pool does not affect the integrity of the pool, but might have an impact on read performance, depending on the workload and the ratio of dataset size to cache size. 
+Note you cannot share dedicated L2ARC devices between ZFS pools.
```

Comment:
```suggestion
The web interface always stripes L2ARC instead of mirroring it since the system recreates L2ARC contents at boot. 
If an individual L2ARC pool SSD fails, it will not affect pool integrity, but it might impact read performance depending on the workload and the ratio of dataset size to cache size. 
You cannot share dedicated L2ARC devices between ZFS pools.
```